WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,16 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read("app/views/search_results/wca-regulations.json"))
```

Comment:
I changed it to use rails root and froze the result.

I don't think in the end the file will be in the git repository: it should probably be built with the Regulations, which mean the file might not be here if one did not build them.